words.json

```diff
@@ -217,5 +217,21 @@
             "antidiscrimination",
             "meteorological",
             "telecommunication"
-        ]
+        ],
+        "8":
```

Comment:
Hey,
Move all of these words under the "5" category (That denotes 5 syllable words) instead of creating a new one. I should have been more specific. Thank you :)

Reply:
Sure